<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_editar_dieta</class>
 <widget class="QMainWindow" name="janela_editar_dieta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-size: 16px;
    color: #fff;
}

#_txt_dieta{
	font-size: 26px;
	font-weight: bold;
}

#_txt_usuario{
	font-size: 26px;
	font-weight: bold;
}

#_txt_cafedamanha{
	font-weight: bold;
}

#_txt_lanchedamanha{
	font-weight: bold;
}

#_txt_almoco{
	font-weight: bold;
}

#_txt_lanchedatarde{
	font-weight: bold;
}

#_txt_jantar{
	font-weight: bold;
}

#_txt_ceia{
	font-weight: bold;
}

#_txt_comentario{
	font-weight: bold;
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QPushButton:hover{
    background: #53A3FF;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;
}

QTextEdit {
    background: #4A90E2; /* Cor de fundo semelhante à paleta */
    border: 1px solid #fff; /* Borda branca */
    border-radius: 8px; /* Bordas arredondadas */
    padding: 10px; /* Espaçamento interno */

    font-family: &quot;Segoe UI&quot;; /* Fonte semelhante aos outros elementos */
    font-size: 16px; /* Tamanho da fonte */
    color: #fff; /* Cor do texto branca */
}

QTextEdit:focus {
    background: #53A3FF; /* Cor de fundo ao ganhar foco */
    border: 1px solid #fff; /* Mantém a borda branca */
    border-radius: 8px; /* Mantém bordas arredondadas */
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_txt_dieta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dieta de</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_usuario">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_cafedamanha">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Café da manhã:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_cafedamanha_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_lanchedamanha">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lanche da manhã:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_lanchedamanha_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_almoco_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_almoco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Almoço:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_lanchedatarde">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lanche da tarde:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_lanchedatarde_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_jantar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Jantar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_jantar_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_ceia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ceia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_ceia_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_cafedamanha">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_lanchedamanha">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_almoco">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_lanchedatarde">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_jantar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_ceia">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_plus.png</normaloff>:/img/Imagens/imagem_plus.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_comentario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Observações:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="_te_comentario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>551</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_salvar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_imagem">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="recursos.qrc">:/img/Imagens/imagem_apple.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_cafedamanha_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_lanchedamanha_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_almoco_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_lanchedatarde_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_jantar_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_ceia_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/img/Imagens/imagem_less.png</normaloff>:/img/Imagens/imagem_less.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
